### PR TITLE
dev/EC-27 chevron unit test

### PR DESCRIPTION
**Pull request by IgorKulishov**

### CODE REVIEW

File: src/app/cart/components/placed-orders/placed-orders.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import { TestBed, async, ComponentFixture } from '@angular/core/testing';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
```
Comment (IgorKulishov) [inline src/app/cart/components/placed-orders/placed-orders.component.spec.ts:2]: Please remove unused code

File: src/app/app.component.spec.ts [hunk 2]
```diff
@@ -1,33 +1,60 @@
...
 import { RouterTestingModule } from '@angular/router/testing';
-import { TranslateService } from '@ngx-translate/core';
 import { AppComponent } from './app.component';
-import { TranslateServiceMock, StoreMock } from '../test/mock';
+import { TranslateServiceMock, StoreMock, initialAppState } from '../test/mock';
 import { Store } from '@ngrx/store';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { TranslateLoader, TranslateModule, TranslateService } from '@ngx-translate/core';
+import {Observable} from 'rxjs';
+import { By } from '@angular/platform-browser';
```
Comment (g1er) [inline src/app/app.component.spec.ts:10]: Pls delete _By_ as unused

File: src/app/cart/components/placed-orders/placed-orders.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+import {TranslateServiceMock, StoreMock, cartReducerStateMock, TranslateLoaderMock} from '../../../../test/mock';
+import { Store } from '@ngrx/store';
+import { AppStates } from '../../store/states/cart.states';
+import { BsModalService, ModalDirective } from 'ngx-bootstrap/modal';
+import { ModalModule } from 'ngx-bootstrap/modal';
+import { CartService } from '../../../core/services/cart.service';
+import { provideMockStore, MockStore } from '@ngrx/store/testing';
+import { HttpClientTestingModule} from '@angular/common/http/testing';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { By } from '@angular/platform-browser';
```
Comment (g1er) [inline src/app/cart/components/placed-orders/placed-orders.component.spec.ts:15]: Pls delete _By_ as unused

**Review by g1er — APPROVED** [review]

File: src/app/app.component.spec.ts (hunk 2 above)
Comment (IgorKulishov) [reply src/app/app.component.spec.ts:10]: Thank you! I will remove it

File: src/app/cart/components/placed-orders/placed-orders.component.spec.ts (hunk 3 above)
Comment (IgorKulishov) [reply src/app/cart/components/placed-orders/placed-orders.component.spec.ts:15]: Thank you, I will remove it

**Pull request merged**